community-resources/templates/metric-template.md

```diff
@@ -44,6 +44,8 @@ List of people who would like to be mentioned as contributors to this metric
 
 ***This metric was last reviewed on [date] as part of [ release version] [or recurring review process]***
 
+### To reference this metric, please use the static URL: https://chaoss.community/?p=[PageID]
```

Comment:
I would suggest to not make it a heading.

```suggestion
To reference this metric, please use the static URL: https://chaoss.community/?p=[PageID]
```

Reply:
It may be missed at the bottom of the page. 

Do we want it to be subtle or highly visible? I am fine either way.

Reply:
I think this might require a little more explanation. As it's written right now, it's not clear how that [PageID] gets assigned. I could see new people developing metrics trying to fill that out by assigning a PageID themselves, which could cause confusion later. If we put this in the metric template, I think we need to be clear that the person filling out the rest of the template should not change that line because the PageID will be assigned later by the release team? The other option is to leave this out of the template, since it needs to be done later after release anyway.

Reply:
I suppose I would lean towards more explanation rather than leaving it out. I think it is important to have a clear idea of the specifications for a completed metrics document (even if the doc is completed in stages).

Reply:
My preference is to actually leave it out of the template. We talked in the website meeting that we could simply check/audit for permalinks on a quarterly basis. As we track them (permalinks) in the spreadsheet, it should be pretty easy for website WG folks to identify and add any new links.

Reply:
Note for clarity:

- Permalinks are human-readable URLs.
- Page Ids are the numerical URLs.

I did add a column in the spreadsheet to track Page IDs.

I would err on the side of well-documented and transparent processes, but I also understand we don't want to add too much complexity to the metrics document.

Reply:
I see the value of not having it in the template for simplicity when creating new issues. The release team could have a checklist item to add that information on the markdown file when setting up the WordPress page.

However the notice gets added to Markdown in GitHub, I believe it is valuable to include it in the markdown file so that anyone who comes to the metric on GitHub finds their way to the official and published metric page.

Reply:
> It may be missed at the bottom of the page.
> 
> Do we want it to be subtle or highly visible? I am fine either way.

+1 subtle because it is not part of the metric definition but rather an administrative information that only a few people will care about